app-text/sigil/sigil-0.9.12.ebuild

```diff
@@ -0,0 +1,82 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{4,5,6,7} )
+
+inherit cmake-utils python-single-r1
+
+my_pn="Sigil"
```

Comment:
Global variables are written with upcase lettering, you should fix this and update the variable used through this ebuild.